Common/src/config_structure.cpp

```diff
@@ -3656,6 +3671,20 @@ void CConfig::SetPostprocessing(unsigned short val_software, unsigned short val_
   	}
   }
 
+  /* --- Throw error if UQ used for any turbulence model other that SST --- */
+
+  if (Kind_Solver == RANS && Kind_Turb_Model != SST && using_uq){
+      cout << "UQ capabilities only implemented for SST turbulence model" << endl;
+      exit(EXIT_FAILURE);
```

Comment:
Please switch these to use the SU2_MPI::Error() routine instead of exit(). There are examples in this routine.

Reply:
Fixed it! Thanks, I didn't realize we had shifted to that error routine.